src/pages/home/report/ReportActionsView.js

```diff
@@ -373,7 +373,7 @@ class ReportActionsView extends React.Component {
                 renderItem={this.renderItem}
                 CellRendererComponent={this.renderCell}
                 contentContainerStyle={[styles.chatContentScrollView]}
-                keyExtractor={item => `${item.action.sequenceNumber}`}
+                keyExtractor={item => `${item.action.timestamp}`}
```

Comment:
Sadly, this isn't actually going to work out. It turns out, it's quite easy to hit a problem if you send messages quickly (more than one per second), since `moment().unix()` (the code creating this timestamp, in `src/libs/actions/Report.js`) easily becomes non-unique.

Instead, I'm thinking we can change this to `item.action.clientID` since this comes from `optimisticReportActionID` in `Report.js`, which is created with `Date.now()` (more digits than `moment().unix()` plus a random number (see https://github.com/Expensify/Expensify.cash/pull/1394/files where this was added).

From my investigation, this does still a slight issue where `.clientID` doesn't exist when initially merging the new action into Onyx (in `Report.js -> addAction`) since we don't set `clientID` there -> `clientID` is added in the server code later, which is how it comes back to the front-end code. 

To fix this, I think we can just add `clientID: optimisticReportActionID,` when updating Onyx locally. Please let me know your thoughts on this 👍

Reply:
Ah, yeah it seems to work with `clientID` as well. updating the pr